tuf-spec.md

```diff
@@ -1074,42 +1074,50 @@ repo](https://github.com/theupdateframework/specification/issues).
+  workflow maximum duration T to the recorded update start time to derive the
+  fixed update expiration time. The value for T is set by the authors of the
+  application using TUF. For example, T may be tens of minutes.
+  This update expiration time will be used when checking for freeze attacks,
+  and is fixed at the beginning of the update workflow to prevent metadata
+  from expiring during an in-progress update.
+
+  **5.1**. **Load the trusted root metadata file.** We assume that a good,
   trusted copy of this file was shipped with the package manager or software
   updater using an out-of-band process.  Note that the expiration of the
   trusted root metadata file does not matter, because we will attempt to update
   it in the next step.
 
-  **5.1**. **Update the root metadata file.** Since it may now be signed using
+  **5.2**. **Update the root metadata file.** Since it may now be signed using
   entirely different keys, the client MUST somehow be able to establish a
   trusted line of continuity to the latest set of keys (see Section 6.1). To do
   so, the client MUST download intermediate root metadata files, until the
   latest available one is reached. Therefore, it MUST temporarily turn on
   consistent snapshots in order to download _versioned_ root metadata files as
   described next.
 
-  * **5.1.1**. Let N denote the version number of the trusted root metadata
+  * **5.2.1**. Let N denote the version number of the trusted root metadata
   file.
 
-  * **5.1.2**. **Try downloading version N+1 of the root metadata file**, up to
+  * **5.2.2**. **Try downloading version N+1 of the root metadata file**, up to
   some W number of bytes (because the size is unknown). The value for W is set
   by the authors of the application using TUF. For example, W may be tens of
   kilobytes. The filename used to download the root metadata file is of the
   fixed form VERSION_NUMBER.FILENAME.EXT (e.g., 42.root.json). If this file is
   not available, or we have downloaded more than Y number of root metadata
-  files (because the exact number is as yet unknown), then go to step 5.1.9.
+  files (because the exact number is as yet unknown), then go to step 5.2.9.
   The value for Y is set by the authors of the application using TUF. For
   example, Y may be 2^10.
 
-  * **5.1.3. Check for an arbitrary software attack.** Version N+1 of the root
+  * **5.2.3. Check for an arbitrary software attack.** Version N+1 of the root
   metadata file MUST have been signed by: (1) a threshold of keys specified in
   the trusted root metadata file (version N), and (2) a threshold of keys
   specified in the new root metadata file being validated (version N+1).  If
   version N+1 is not signed as required, discard it, abort the update cycle,
   and report the signature failure.  On the next update cycle, begin at step
-  5.0 and version N of the root metadata file.
+  5.1 and version N of the root metadata file.
```

Comment:
There are two more "On the next update cycle, begin at step 5.0" below. Were they not changed for a reason?

Reply:
They were not. I thought I'd caught all of the cross-references. 🤦